sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/PermissionCrudTest.java

```diff
@@ -44,15 +45,18 @@ public void createPermission() throws Exception {
         CosmosPermissionProperties permissionSettings = new CosmosPermissionProperties()
                 .setId(UUID.randomUUID().toString())
                 .setPermissionMode(PermissionMode.READ)
-                .setResourceLink("dbs/AQAAAA==/colls/AQAAAJ0fgTc=");
+                .setContainerName("AQAAAJ0fgTc=")
```

Comment:
this is called `setContainerName` but in the test `.setContainerName("AQAAAJ0fgTc=")` uses portion of the selflink.
So is this the container name or just a portion of the selflink?

Reply:
it's the container name... I will make a change to make it more obvious; initially I did not want to mess with the test used/expected values.